export.py

```diff
@@ -640,7 +640,7 @@ def create_mesh(ob, scene):
     # elif is_subd_displace_last(ob):
     #    ob.modifiers[len(ob.modifiers)-2].show_render = False
     #    ob.modifiers[len(ob.modifiers)-1].show_render = False
-    mesh = ob.to_mesh(scene, True, 'RENDER', calc_tessface=True,
+    mesh = ob.to_mesh(scene, True, 'RENDER', calc_tessface=False,
```

Comment:
Weird.  Well that's a huge change, basically I think blender was exporting the subdivided faces to the addon and then we were exporting them as a subdiv.  Which is ridiculous!